pom.xml

```diff
@@ -724,66 +728,48 @@
                 <relocation>
-                  <pattern>net.minidev</pattern>
-                  <shadedPattern>${shadeBase}.net.minidev</shadedPattern>
+                  <pattern>javax</pattern>
+                  <shadedPattern>${shadeBase}.javax</shadedPattern>
                 </relocation>
                 <relocation>
-                  <pattern>org.objectweb</pattern>
-                  <shadedPattern>${shadeBase}.org.objectweb</shadedPattern>
+                  <pattern>jersey</pattern>
+                  <shadedPattern>${shadeBase}.jersey</shadedPattern>
+                </relocation>
+                <relocation>
+                  <pattern>net</pattern>
+                  <shadedPattern>${shadeBase}.net</shadedPattern>
+                  <excludes>
+                    <exclude>net.snowflake.ingest.**</exclude>
+                  </excludes>
                 </relocation>
                 <relocation>
-                  <pattern>com.fasterxml</pattern>
-                  <shadedPattern>${shadeBase}.fasterxml</shadedPattern>
+                  <pattern>org</pattern>
+                  <shadedPattern>${shadeBase}.org</shadedPattern>
                 </relocation>
                 <relocation>
-                  <pattern>org.apache</pattern>
-                  <shadedPattern>${shadeBase}.apache</shadedPattern>
+                  <!--
+                    this is the parquet shade location
+                    https://github.com/apache/parquet-mr/blob/master/pom.xml#LL72C6-L72C18
+                  -->
+                  <pattern>shaded.parquet</pattern>
+                  <shadedPattern>${shadeBase}.shaded.parquet</shadedPattern>
                 </relocation>
               </relocations>
-              <filters>
-                <filter>
-                  <artifact>*:*</artifact>
-                  <excludes>
-                    <exclude>META-INF/LICENSE*</exclude>
-                    <exclude>META-INF/NOTICE*</exclude>
-                    <exclude>META-INF/DEPENDENCIES</exclude>
-                    <exclude>META-INF/maven/**</exclude>
-                    <exclude>META-INF/services/com.fasterxml.*</exclude>
-                    <exclude>META-INF/*.xml</exclude>
-                    <exclude>META-INF/*.SF</exclude>
-                    <exclude>META-INF/*.DSA</exclude>
-                    <exclude>META-INF/*.RSA</exclude>
-                  </excludes>
-                </filter>
-                <filter>
-                  <artifact>commons-logging:commons-logging</artifact>
-                  <excludes>
-                    <exclude>org/apache/commons/logging/impl/AvalonLogger.class</exclude>
-                  </excludes>
-                </filter>
-                <filter>
-                  <artifact>org.slf4j:slf4j-simple</artifact>
-                  <excludes>
-                    <exclude>org/slf4j/**</exclude>
-                  </excludes>
-                </filter>
-              </filters>
```

Comment:
It's not obvious if any of this needs to stay but it seemed like it might be able to go.